### ISSUE TITLE
Project page linkedin and email icons image link broken

### ISSUE THREAD

**Issue by mthernandez** [mthernandez/mthernandez.github.io#18]
Icons for LinkedIn and Email links showing alt text, link to images likely broken (missing ../../?). Consider fixing with #16 and/or #2
